src/clients/c++/request.h

```diff
@@ -666,6 +667,29 @@ class InferContext {
   Error GetReadyAsyncRequest(
       std::shared_ptr<Request>* async_request, bool wait);
 
+  /// Send current request through the context's stream. All requests sent will
```

Comment:
Don't say anything about being directed to the same model instance... since that is beyond the scope of what the grpc stream itself will do.